tony-portal/app/views/tableMetadata.scala.html

```diff
@@ -44,25 +63,59 @@
       </tr>
   }
   </tbody>
+    <tfoot>
+      <tr>
+        <th>App ID</th>
+        <th>Configuration</th>
+        <th>Started</th>
+        <th>Completed</th>
+        <th>User</th>
+        <th>Status</th>
+        <th class = "rmv_elmt">ResourceManager Link</th>
```

Comment:
Can you add a comment explaining what "rmv_elmt" is for?

Reply:
@erwa added comment, please take a look